src/apiCalls.js

```diff
@@ -0,0 +1,12 @@
+export const postForm = (newPost: {}) => {
+  return fetch(`http://localhost:5000/api/v1/posts`, {
```

Comment:
Thanks for creating the ApiCalls file. I'll move our initial fetch over here. I'd love to see if we can change the endpoint and post to the Heroku server but I'm not sure if that's possible